### DISCUSSION TITLE
Promitor Scraper Chart v2.6.0

### DISCUSSION THREAD

**Discussion by tomkerkhove** [promitor/charts#62] [Announcements]
*Chart: v2.6.0 | App: v2.5.0*

## TL;DR
```shell
helm repo add promitor https://charts.promitor.io/
helm install promitor-agent-scraper promitor/promitor-agent-scraper
```

### Features

- Provide scraper for Azure Data Factory ([docs](https://docs.promitor.io/configuration/v2.x/metrics/data-factory) | [#359](https://github.com/tomkerkhove/promitor/issues/359))
- Provide scraper for Azure Data Share ([docs](https://docs.promitor.io/configuration/v2.x/metrics/data-share) | [#1678](https://github.com/tomkerkhove/promitor/issues/1678))
- Provide system metrics related to agent performance & resources ([docs](https://docs.promitor.io/operations/#performance) | [#341](https://github.com/tomkerkhove/promitor/issues/341))
- Provide system metrics indicating ARM throttling status ([docs](https://docs.promitor.io/operations/#azure-resource-manager-api---consumption--throttling) | [#1738](https://github.com/tomkerkhove/promitor/issues/1738))

### Fixes / Changes

None.

### Breaking Changes

None.

### Removal

None.

<hr /><em>This discussion was created from the release <a href='https://github.com/promitor/charts/releases/tag/Scraper-v2.6.0'>Promitor Scraper Chart v2.6.0</a>.</em>
